src/ReactWrapper.js

```diff
@@ -438,6 +438,17 @@ export default class ReactWrapper {
   }
 
   /**
+   * Returns a new wrapper with a specific child
+   *
+   * @param {Number} [index]
+   * @returns {ReactWrapper}
+   */
+  childAt(index) {
+    const allChildren = this.flatMap(n => childrenOfInst(n.node));
+    return allChildren.at(index);
```

Comment:
i think that might be overcomplicating the childAt API - if you want a selector i think `.children(selector).at(index)` is probably better.